src/components/filter-pane/one-of-filter-shelf.tsx

```diff
@@ -95,7 +95,15 @@ export class OneOfFilterShelfBase extends React.PureComponent<OneOfFilterShelfPr
 
   private onSelectOne(value: string | number | boolean | DateTime) {
     const {index} = this.props;
-    this.filterModifyOneOf(index, [value]);
+    if (typeof value === 'string') {
```

Comment:
This has negative runtime effect. Please remove and make the types of this.filterModifyOneOf more general.

Reply:
Fixed